games/src/main/scala/com/gu/game_review_parser/parsers/StandardParser.scala

```diff
@@ -0,0 +1,117 @@
+package com.gu.game_review_parser.parsers
+
+import com.gu.contentapi.client.model.v1.Content
+import com.gu.game_review_parser.ParsedGameReview
+import org.jsoup.Jsoup
+
+import scala.util.Try
+
+object StandardParser {
+  def parseContent(content: Content): Option[ParsedGameReview] = {
+    val parsed = for {
+      title <- getTitle(content.webTitle)
```

Comment:
What does the title represent? The name of the game? If so it should ONLY be the name of the game itself, not with any of the fluff surrounding it.